bom/application/pom.xml

```diff
@@ -151,7 +151,7 @@
         <kotlin.version>1.7.20</kotlin.version>
         <kotlin.coroutine.version>1.6.4</kotlin.coroutine.version>
         <kotlin-serialization.version>1.4.1</kotlin-serialization.version>
-        <kubernetes-client.version>6.1.1</kubernetes-client.version> <!-- Please check with Java Operator SDK team before updating -->
+        <kubernetes-client.version>6.2.0</kubernetes-client.version> <!-- Please check with Java Operator SDK team before updating -->
```

Comment:
Sorry you will have to rebase as I just merged a Dekorate bump. I tried to do it but I can't push to your fork because you have this weird org setup :).

Reply:
No worries, Done :)

> you have this weird org setup :).

Yes, I'm using this to better organize the repos in my profile. However, last month I realized (Ioannis shared this) that this won't work well for PRs since I can't let maintainers access my branches :( If someone knows of a better alternative, please share ;)